games-strategy/0ad/0ad-0.0.22.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="O A.D. - Game"
+DESCRIPTION="A free, open-source, Real-Time Strategy (RTS) game."
+HOMEPAGE="https://play0ad.com"
+COPYRIGHT="Wildfire Games"
+LICENSE="0ADLicense"
+REVISION="1"
+SOURCE_URI="https://releases.wildfiregames.com/0ad-0.0.22-alpha-unix-build.tar.gz"
```

Comment:
```diff
-SOURCE_URI="https://releases.wildfiregames.com/0ad-0.0.22-alpha-unix-build.tar.gz"
+SOURCE_URI="https://releases.wildfiregames.com/0ad-$portVersion-alpha-unix-build.tar.gz"
```
Hmm, Having this "alpha" string detached of the version is not nice.
If you rename the recipe to 0ad-0.0.2~alpha.recipe then you can use:
```
SOURCE_URI="https://releases.wildfiregames.com/0ad-${portVersion/\~/-}-unix-build.tar.gz"
CHECKSUM_SHA256="..."
SOURCE_DIR="0ad-${portVersion/\~/-}"
PATCHES="0ad-${portVersion/\~/-}.patch"
```
This is just a suggestion, but I think it would be a good idea.
BTW, if you buy this, you can wait until you're almost done with this recipe.